### DISCUSSION TITLE
Connect NodeJS error console to Github actions

### DISCUSSION THREAD

**Discussion by anusheel** [The-Black-Spatula-Project/black-spatula-project#5] [Ideas]
This will provide visibility into any errors that happen in the pipeline execution to the Github Actions UI. Currently this communication is via stdout so either we use a structured format and stick to stdout or think of some other way

**Comment by fahminlb33** [comment]
Hi, Actions has support for this via the [Workflow commands](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions?tool=bash#setting-an-error-message). I can create a PR for this.
